src/stores/icon-customizations.store.ts

```
@@ -2,253 +2,269 @@ import { writable, get } from "svelte/store";
-        }),
+				state.svgContent = cleanedSvg;
 
-        /** @deprecated A way to handle image icons will be implemented */
-        selectImageIcon: (url: string) => update(state => {
-            if (!state) return state;
+				return state;
+			}),
 
-            verboseMode && console.log("Setting image icon");
+		/** @deprecated A way to handle image icons will be implemented */
+		selectImageIcon: (url: string) =>
```

ESLint

[error] 'url' is defined but never used.

(@typescript-eslint/no-unused-vars)